package.json

```diff
@@ -18,7 +19,6 @@
     "@types/react-dom": "^18.0.11",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
-    "react-scripts": "5.0.1",
```

Comment:
> Oh, I forgot to ask - does Storybook stay mad if this is in devDependencies? Or only during the install process?

😮 You're right! This only bit during the install. It runs fine now. I'll move it back, excellent catch.

> > Arguably, more of our packages should be moved here, but meanwhile we should at least move this one.
> 
> What else are you thinking? Or, are you saying we shouldn't be using `devDependencies` at all in a front-end web app like this? I'm receptive to either!

This was mostly a reference to our conversation. I prefer to organize them like you said, separating packages used during development (`devDependencies`) from those intended to be used in the final build (`dependencies`). But I'm unsure there's any practical benefit to that, and evidently it can cause a few speedbumps.

I don't think we need to change anything now, but I remain receptive to jettisoning `devDependencies` if it ever causes problems.

For now, since I'm going to remove this commit from the branch, this editorializing will be removed anyway, lol.